monasca/values.yaml

```diff
@@ -483,8 +509,8 @@ alarms:
   name: alarms
   enabled: true
   image:
-    repository: monasca/alarms
-    tag: 1.1.1
+    repository: aadams/monasca-alarms
```

Comment:
This should use monasca/alarms (we might need to push a new version). If that hasn't merged yet we can use the new image in our override values and update versions later.

Reply:
good catch, that was an accident